src/plugins/controls/public/range_slider/embeddable/range_slider_embeddable.tsx

```diff
@@ -168,24 +168,29 @@ export class RangeSliderEmbeddable
 
     // fetch available min/max when input changes
     this.subscriptions.add(
-      dataFetchPipe.subscribe(async (changes) => {
-        try {
-          await this.runRangeSliderQuery();
-          await this.buildFilter();
-        } catch (e) {
-          this.onLoadingError(e.message);
-        }
-      })
+      dataFetchPipe
+        .pipe(
+          mergeMap(async (changes) => {
```

Comment:
Again, I think a `switchMap` might be better here.

Reply:
Addressed in [`0fab35e` (#169522)](https://github.com/elastic/kibana/pull/169522/commits/0fab35e43a6392ec41e358d0cd95140323fac58a)